psi4/driver/qcdb/libmintsgshell.py

```diff
@@ -193,7 +188,7 @@ def __init__(self, am, c, e, pure, nc, center, start, pt='Normalized', rpowers=N
             self.normalize_shell()
             self.erd_normalize_shell()
         else:
-            self.PYerd_coef = [0.0 for i in c]
+            self.PYerd_coef = [0.0] * self.nprimitive() 
```

Comment:
The `[..]*integer` version here makes a list of references to the same float object, which has different consequences for mutable vs. immutable types.  The list comprehension `[ 0.0 for i in c]` creates distinct objects, removing the need to know whether the type is immutable or not, which is why I tend to prefer that version.  In this case both are safe, as the float is immutable, so it's fine to use either.  I'm commenting here to point out this difference for future reference, and to solicit for opinions from other devs who know Python better than I, in case what I said is wrong.